components/icons/hamburger.js

```diff
@@ -16,6 +16,13 @@ const Hamburger = () => (
         width: 26px;
         height: 26px;
       }
+
+      @media screen and (min-width: 30em) and (max-width: 60em){
```

Comment:
Just a heads up, our goal to is to use `rem` for any sizing unless you have specific reasonings for using `em` and `px` here. If not, do you mind updating these sizes to the use `rem`? Feel free to update any `px` or `em` sizing you come across if you feel like it.

Reply:
yeah I can change those.  I used dustin.haefele@gmail.com because that's what Tachyons uses for media queries but I will change them to rem it shouldn't cause a problem.  All of the px were there already but I can update to rem real quick and push back up

Reply:
it's updated